package.json

```diff
@@ -228,7 +240,8 @@
     "@stryker-mutator/html-reporter": "^3.0.1",
     "@stryker-mutator/karma-runner": "^3.0.1",
     "@stryker-mutator/mocha-runner": "^3.0.1",
-    "@stryker-mutator/typescript": "^4.0.0"
+    "@stryker-mutator/typescript": "^4.0.0",
+    "typescript": "~4.9.5"
```

Comment:
Do we need both this a L230, or can we just use one (either upgrade the version on 230, or only have the resolution)?

Reply:
Done